src/components/Books.module.css

```diff
@@ -1,4 +1,81 @@
 .book {
   display: flex;
   flex-direction: column;
+  width: 75rem;
+  height: 10.625rem;
+  margin: 2.313rem 6.25rem 0;
+  padding: 2rem 9.188rem 1.625rem 1.688rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+}
```

Comment:
- [ ] Please work on centering your app contents to the page just as the original design from the header to the form. Currently, your app contents are aligned to the left and there is too much space from the end of the book cards and the user icon to the right end of the screen. Please center all your content just as I marked them in the screenshot 👍 

**The original Design**
![image](https://user-images.githubusercontent.com/50941074/193430878-a339c7ac-5f5b-4c3f-89ef-2cad08d30259.png)

**Your Design**
![image](https://user-images.githubusercontent.com/50941074/193431142-c3fd64b8-ca5b-479c-9817-6673bd774ad3.png)